modules/wowchemy/layouts/partials/blocks/people.html

```diff
@@ -0,0 +1,74 @@
+{{/* Wowchemy Blocks: People */}}
+{{/* Documentation: https://wowchemy.com/blocks/ */}}
+{{/* License: https://github.com/wowchemy/wowchemy-hugo-themes/blob/main/LICENSE.md */}}
+
+{{/* Initialise */}}
+{{ $page := .wcPage }}
+{{ $block := .wcBlock }}
+{{ $show_social := $block.design.show_social | default false }}
+{{ $show_interests := $block.design.show_interests | default true }}
+{{ $show_organizations := $block.design.show_organizations | default false }}
+{{ $show_role := $block.design.show_role | default true }}
+
+<div class="row justify-content-center people-widget">
+  {{ with $block.content.title }}
+    <div class="section-heading col-12 mb-3 text-center">
+      <h1 class="mb-0">{{ . | markdownify | emojify }}</h1>
+      {{ with $block.content.subtitle }}<p class="mt-1">{{ . | markdownify | emojify }}</p>{{ end }}
+    </div>
+  {{ end }}
+
+  {{ with $block.content.text }}
+    <div class="col-md-12">
+      {{ . | emojify | $page.RenderString }}
+    </div>
+  {{ end }}
+
+  {{ range $block.content.user_groups }}
+    {{ $query := where (where site.Pages "Section" "authors") ".Params.user_groups" "intersect" (slice .) }}
+
+    {{/* Sort */}}
+    {{ $sort_by := $block.content.sort_by | default "last_name" }}
+    {{ if not ( in "last_name first_name title" $sort_by ) }} {{ $sort_by = "last_name" }} {{ end }}
+    {{ $sort_by = printf ".Params.%s" $sort_by }}
```

Comment:
If we prefix `.Params.` to the `sort_by` automatically, we should probably do the same for any other uses of the sort_by option in Wowchemy, such as in the Collections (and Portfolio?) blocks, otherwise different behaviours will be very confusing to users.

Reply:
Ok, I also remove the automatic `.Params.` prefix to align this block to the others.
Although, I think maybe it could be nice to have an autoprefix just if `last_name`, `first_name` or `title` are passed as parameters, since these are the most common options to sort the people.

It could be something like:
```
{{ if in "last_name first_name title" $sort_by }} {{ $sort_by = printf ".Params.%s" $sort_by }}  {{ end }}
```

What do you think? Otherwise, I'll just remove it, no problem!

Reply:
If the first letter of the sort_by param is capitalised then it's a built-in Hugo param right, otherwise it's a parameter which requires prefixing with `Params.`?

I have also observed that users find this Hugo behaviour very confusing and prone to error.

So I suggest we check if first letter of sort_by is capitalised, if it is then pass it as it is, otherwise prefix it. If this works successfully, then let's apply this logic to all instances where sort_by is used.

Reply:
I need to think about this. In principle, I think nothing stops me to create a custom parameter (eg. `LastName` instead of `last_name`), so I don't know if this logic can actually be applied.

If you agree, we can close this PR (which I'll immediately follow with a new one to fix the Wowchemy version in `go.mod`) as it is, so that users need to input `.Params.last_name` to sort authors by their last name. 

Then, I'll open another issue to discuss and investigate about your proposed solution to discover built-in or custom parameters.

Reply:
@Agos95 the Wowchemy parameters are all standardised to lowercase-underscore convention, so that is fine, we can go ahead and make things easier for the user based on the above comment at https://github.com/wowchemy/wowchemy-hugo-themes/pull/2914#discussion_r1111261900

Reply:
Ok, I added the function in the `wowchemy-core` module, since it is independent on bootstrap, and I tested it locally.
I also add it in `people`, `collection` and `portfolio` blocks as comments. This is due to the fact that the version of `wowchemy-core` must be updated (in `modules/wowchemy/go.mod`) in order to detect the new function, and I don't have any idea on what I should do.

Maybe the best thing is to merge this PR, create a new tag/version for `wowchemy-core`, and then I can follow with another PR to fix the versions. But I wait for your instructions, since I don't have any experience on this.

Reply:
See https://github.com/wowchemy/wowchemy-hugo-themes/pull/2914#issuecomment-1505706541

In order to merge this PR without splitting it into separate PRs for each module change, we will go ahead with the latter suggestion you made.